source/user-manual/agents/remote-upgrading/upgrading-agent.rst

```diff
@@ -12,6 +12,10 @@ Upgrading an agent remotely can be performed at the command line and through the
 Using the command line
 ----------------------
 
+.. note::
+  The agent_upgrade tool is available since Wazuh version 3.0. It can only be used to upgrade agents with an actual version equal to or greater than 3.0.
+  It is not available for Windows XP agents.
```

Comment:
Let's write in positive:
> Remote upgrades are compatible with agents running on Windows 6.0 (Vista and Server 2008) and newer.

Reply:
Sorry for the delay, the changed is already done.